reactor-core/src/main/java/reactor/core/publisher/ConnectableFluxStats.java

```diff
@@ -0,0 +1,107 @@
+package reactor.core.publisher;
+
+import java.util.function.Consumer;
+
+import org.reactivestreams.Publisher;
+import reactor.core.CoreSubscriber;
+import reactor.core.Disposable;
+import reactor.core.Fuseable;
+import reactor.core.Scannable;
+import reactor.util.stats.StatsMarker;
+import reactor.util.stats.StatsReporter;
+
+public class ConnectableFluxStats<T> extends InternalConnectableFluxOperator<T, T> {
```

Comment:
can this and other classes be package-private?

Reply:
fixed